src/components/elements/roadmap.tsx

```diff
@@ -63,23 +73,16 @@ const StyledButton = styled(LinkButton)`
     z-index: 3;
 `
 
-type RoadmapProps = {
-    portal?: {
-        paragraph?: string | ReactElement
-        frame?: string
-        link?: string
-    }
-}
 const Roadmap = ({ portal }: RoadmapProps) => {
     return (
         <StyledSectionContainer>
             <Container>
                 <Flex width="1202px" fd="column" ai="center" jc="center">
                     <StyledHeader as="h2" type="heading-2" align="center">
-                        <Localize translate_text="What’s next?" />
+                        <Localize translate_text="_t_What’s next?_t_" />
```

Comment:
Did we decide to store short strings directly inside `translation_text` prop?

Reply:
@mohammad-hashemi-deriv we changed the key extractions, right now we can directly passing the texts to the `localize` component and function without defining a variable for them!